### PR TITLE
Setup script tweaks

### PR DESCRIPTION
**Pull request by dacook**
#### What? Why?

Reduces unnecessary Bundler output, and speeds up the script.

Screen Shot 2023-08-09 at 10 24 14 am

BTW This handy script can be run at any time to prepare your environment after pulling the latest changes on master, including gems, node packages and database migrations.

#### What should we test?

If you have already installed the latest gems today, you can first remove one to test that it gets installed again. Eg:
```
gem uninstall ransack # randomly chosen gem
bin/setup
```

#### Release notes

Changelog Category: Technical changes

The title of the pull request will be included in the release notes.

### CODE REVIEW

File: bin/setup [hunk 1]
```diff
@@ -27,11 +27,8 @@ FileUtils.chdir APP_ROOT do
   #   FileUtils.cp "config/database.yml.sample", "config/database.yml"
   # end
 
-  puts "\n== Preparing database =="
-  system! "bin/rails db:prepare"
-
-  puts "\n== Removing old logs and tempfiles =="
-  system! "bin/rails log:clear tmp:clear"
+  puts "\n== Preparing database, removing old logs and tempfiles =="
+  system! "bin/rails db:prepare log:clear tmp:clear"
```
Comment (mkllnk) [inline bin/setup:31]: I would have expected Spring to skip the boot time but I did observe this slowness as well. I wonder why...

File: bin/setup [hunk 2]
```diff
@@ -4,6 +4,8 @@ require "fileutils"
 # path to your application root.
 APP_ROOT = File.expand_path("..", __dir__)
 
+BUNDLE_ENV = { "BUNDLE_SUPPRESS_INSTALL_USING_MESSAGES"=> "true" }
```
Comment (mkllnk) [inline bin/setup:7]:
```suggestion
BUNDLE_ENV = { "BUNDLE_SUPPRESS_INSTALL_USING_MESSAGES" => "true" }
```

**Review by mkllnk — APPROVED** [review]
Great, thank you!

**Comment by mkllnk** [comment]
Works on my machine! Test passed.

I think I hesitated changing these things in the past because whenever we update Rails it may suggest a newer version of this script. And the more changes there are to the original, the more we have to review to see if the new version has any improvements compared to our version. But rails upgrades are quite rare...

**Comment by dacook** [comment]
> whenever we update Rails it may suggest a newer version of this script

Oh yeah, I forgot about that. For the small benefit it may not be worth the extra maintenance. 
🤷  Not big difference either way I guess.

**Comment by jibees** [comment]
> BTW This handy script can be run at any time to prepare your environment after pulling the latest changes on master, including gems, node packages and database migrations.

Oh nice! Thanks!

**Pull request merged**